build.rs

```diff
@@ -37,6 +37,8 @@ fn gl_generate() {
                 "EGL_EXT_image_dma_buf_import",
                 "EGL_EXT_image_dma_buf_import_modifiers",
                 "EGL_MESA_image_dma_buf_export",
+                #[cfg(target_os = "android")]
+                "EGL_KHR_platform_android",
```

Comment:
We don't actually need to guard the extension generation since many listed extensions already are always generated.

Reply:
Aah okayy!

Reply:
Okayy fixed!!